pages/docs/how-to-guides/events.mdx

```diff
@@ -77,11 +77,6 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
 
 4. Now you can commit your file and create a Pull Request. For more details please see <Link href="/docs/how-to-guides/editing">Editing Guide</Link>
    If you want to know how to properly add start, end, and CFP Close times, feel free to check [Unix Time stamp](https://www.unixtimestamp.com/).
-   export default ({ children }) => (
-   <DocsLayout
-    title="LinkFree Events Documentation"
-    section="Customising your Profile"
-    name="Events">
-    {children}
-    </DocsLayout>
-    );
+   
+   export default ({ children }) => (<DocsLayout  title="LinkFree Events Documentation" section="Customising your Profile" name="Events"> {children}</DocsLayout>);
```

Comment:
Hi Chinmay,  I add the space to code but when I go to my repo it does not changed. It show  like this 👇🏿
![code not change](https://github.com/EddieHubCommunity/LinkFree/assets/89390268/f209e5dc-078e-49a4-b437-5edc9b0cb545)